.travis.yml

```diff
@@ -30,7 +30,7 @@ env:
         - MAIN_CMD='python setup.py'
         - SETUP_CMD='test'
         - EVENT_TYPE='pull_request push'
-        - PIP_DEPENDENCIES=''
+        - PIP_DEPENDENCIES='gwcs'
```

Comment:
It fails because of conflicting dependencies:
https://travis-ci.org/astropy/photutils/jobs/209395861#L463